src/components/cc-stretch/cc-stretch.js

```diff
@@ -0,0 +1,101 @@
+import { css, html, LitElement } from 'lit';
+
+/**
+ * A layout component used to switch visibility between several elements while preventing layout shifts.
+ *
+ * Use `visibleElementId` prop to provide the `id` value corresponding to the element you want to make visible.
+ * Other elements are hidden with `visibility: hidden`.
+ *
+ * More information about the issues this component solves and the technique it uses can be found in [Hubert Sablonnière's blog Post - "Prevent layout shifts with CSS grid stacks"](https://www.hsablonniere.com/prevent-layout-shifts-with-css-grid-stacks--qcj5jo/).
+ *
+ * **Caution**:
+ *
+ * This component relies on a class named `cc-stretch--visible` applied to the relevant slotted element. Make sure you never override this class.
+ *
+ * @cssdisplay inline-grid
+ *
+ * @slot - The containers you want to switch between. The container you want to be displayed must have an `id` attribute matching the value of the `visibleElementId` prop.
+ *
+ * @cssprop {AlignItems} --cc-stretch-align-items - Specify how the items should be distributed / positioned vertically within the grid (Default: `center`. Possible values: https://developer.mozilla.org/en-US/docs/Web/CSS/align-items).
+ * @cssprop {JustifyItems} --cc-stretch-justify-items - Specify how the items should be distributed / positioned horizontally within the grid (Default: `center`. Possible values: https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items).
```

Comment:
I may be biased but I expect the CSS custom properties default value to match the CSS property [initial value](https://developer.mozilla.org/en-US/docs/Web/CSS/initial_value), meaning:
- for `align-items`, the value `normal`
- for `justify-items`, the value... `legacy` ? :thinking: (as stated in [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items#formal_definition))

Reply:
I was thinking the same as you. But my final thought was that we should choose the default values that fit best our use cases. I think it makes sense here to use `center` for both.

Reply:
Makes sense! :+1:

Reply:
About this, I would have asked to remove those and let the user define the properties directly but since we want different defaults, the custom props make sense. It's also better if we want to add another layer later.